src/hub.html

```diff
@@ -155,14 +155,30 @@
                     sticky-object="autoLockOnRelease: true; autoLockOnLoad: true;"
                     position-at-box-shape-border="target:.delete-button"
                     destroy-at-extreme-distances
+                    rotation
                 >
+                    <!-- HACK: rotation component above is required for its side effect of setting YXZ order -->
                     <a-entity class="delete-button" visible-while-frozen>
                         <a-entity mixin="rounded-text-button" remove-networked-object-button position="0 0 0"> </a-entity>
                         <a-entity text=" value:Delete; width:2.5; align:center;" text-raycast-hack position="0 0 0.01"></a-entity>
                     </a-entity>
                 </a-entity>
             </template>
 
+            <template id="paging-toolbar">
+                <a-entity class="paging-toolbar" visible-to-owner>
```

Comment:
As it is currently written yes, since only the owner would network the page state. The alternative would be to making `media-pager` aware of the concept of ownership, but since design wise it seems kind of deisrable to not show the paging buttons for non-owners anyway, it seems better this way.